episodes/what-is-an-image.md

```diff
@@ -0,0 +1,540 @@
+including the microscope optics, detector performance etc.
+
+::::::::::::::::::::::::::::::::::::: discussion 
+
+Read the ['A simple microscope' section of Pete Bankhead's bioimage book](
+https://bioimagebook.github.io/chapters/1-concepts/1-images_and_pixels/images_and_pixels.html#a-simple-microscope).
+
+- What are some factors that influence pixel values?
+- Can you come up with suggestions for any more?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Image dimensions
+
+Let's return to our human mitosis image and explore some of the key features of 
+its image array. First, what size is it?
+
+We can find this out by running the following in Napari's console:
+```python
+
+image.shape
+```
+
+```output
+(512, 512)
+```
+
+The array size (also known as its dimensions) is stored in the `.shape`. Here 
+we see that it is `(512, 512)` meaning this image is 512 pixels high and 512 
+pixels wide. Two values are printed as this image is two dimensional (2D), for 
+a 3D image there would be 3, for a 4D image (e.g. with an additional time 
+series) there would be 4 and so on...
+
+## Image data type
+
+The other key feature of an image array is its 'data type' - this controls which 
+values can be stored inside of it. For example, let's look at the data type for 
+our human mitosis image:
+
+```python
+
+image.dtype
+```
+
+```output
+
+dtype('uint8')
+```
+
+We see that the data type (or 'dtype' for short) is `uint8`. This is short for 
+'unsigned integer 8-bit'. Let's break this down further into two parts - the 
+type (unsigned integer) and the bit-depth (8-bit).
+
+## Type
+
+The type determines what kind of values can be stored in the array, for example:
+
+- Unsigned integer: positive whole numbers
+- Signed integer: positive and negative whole numbers
+- Float: positive and negative numbers with a decimal point e.g. 3.14
+
+For our mitosis image, 'unsigned integer' means that only positive whole numbers 
+can be stored inside. You can see this by hovering over the pixels in the image 
+again in Napari - the pixel value down in the bottom left is always a positive 
+whole number.
+
+## Bit depth
+
+The bit depth determines the range of values that can be stored e.g. only values 
+between 0 and 255. This is directly related to how the array is stored in the 
+computer.
+
+In the computer, each pixel value will ultimately be stored in some binary 
+format as a series of ones and zeros. Each of these ones or zeros is known 
+as a 'bit', and the 'bit depth' is the number of bits used to store each value. 
+For example, our mitosis image uses 8 bits to store each value (i.e. 
+a series of 8 ones or zeros like `00000000`, or `01101101`...).
+
+The reason the bit depth is so important is that it dictates the number of 
+different values that can be stored. In fact it is equal to:
+
+\[\large \text{Number of values} = 2^{\text{bit depth}}\]
+
+Going back to our mitosis image, this means that it can store $2^8 = 256$ 
```

Comment:
```suggestion
Going back to our mitosis image, since it is stored with an integer with a bit-depth of 8, this means that it can store $2^8 = 256$ 
```